src/types/form.ts

```diff
@@ -36,6 +36,19 @@ export enum Colors {
   Grey = 'grey',
 }
 
+export enum Rating {
+  One = 1,
```

Comment:
according to the [docs](https://www.typescriptlang.org/docs/handbook/enums.html), if you were to continue using a numeric enum, you could actually just set `One = 1` and the rest of the numbers would auto-increment